cylc/flow/cfgspec/globalcfg.py

```diff
@@ -1330,6 +1330,28 @@ def default_for(
 
                 .. versionadded:: 8.0.0
             ''')
+
+            with Conf('profile'):
+                Conf('activate', VDR.V_BOOLEAN, True, desc='''
```

Comment:
```suggestion
                Conf('activate', VDR.V_BOOLEAN, False, desc='''
```

Sites like ours should enable the profiler by default in their site configuration. The default, however, should be `False` (note the profiler can only work for certain types of platform anyway).

Reply:
Done